iankduffy/Pro-Coders-Blog
# Added ability to create, view all and view indiviual articles

## Overview

Related Issues:  
https://github.com/iankduffy/Pro-Coders-Blog/issues/2
https://github.com/iankduffy/Pro-Coders-Blog/issues/3
https://github.com/iankduffy/Pro-Coders-Blog/issues/4

## QA

1. Hit localhost:3000/admin this should prompt you to log in with `admin` and `yankee2`
2. Clicking new post will bring up a form to allow you to submit a new form. 
3. Once entered this should take you to the blog post
4. Go to localhost:3000/admin - and you should see the post as a list

## Dependencies
rspec-rails - has been added as the testing suite for the project.
